source_gen/pubspec.yaml

```diff
@@ -6,10 +6,10 @@ repository: https://github.com/dart-lang/source_gen/tree/master/source_gen
 resolution: workspace
 
 environment:
-  sdk: ^3.5.0
+  sdk: ^3.6.0-edge
```

Comment:
Instead of `-edge` we ideally want a dev release, specifically `3.6.0-217.0.dev` it looks like.